lib/axiosHttpClient.ts

```diff
@@ -101,13 +105,14 @@ export class AxiosHttpClient implements HttpClient {
         transformResponse: undefined,
         validateStatus: () => true,
         withCredentials: true,
+        maxContentLength: 1024*1024*1024*10,
```

Comment:
Is this defined in axios? Where does this number come from? I recommend a comment explanation here.